seqr/views/utils/permissions_utils.py

```diff
@@ -34,14 +34,19 @@ def _has_current_policies(user):
 
 
 # User access decorators
+def _require_permission(test_func, error='User has insufficient permission'):
+    def test_user(user):
+        if not test_func(user):
+            raise PermissionDenied(error)
+        return True
+    return test_user
```

Comment:
It looks like renaming `test_func` to `permission_test_func` will be more descriptive. It can remind that not to use this function for other test condition. Otherwise, it could raise `PermissionDenied` for a condition unrelated to permission.